webapp/test/test_repos.py

```diff
@@ -63,3 +67,15 @@ def test_schema(self):
         """Test schema of valid repos API response."""
         response = self.repo.process_list(api_version="v1", data=REPO_JSON)
         assert schemas.repos_schema.validate(response)
+
+    def test_modified_since(self):
+        """Test repos API with 'modified_since' property."""
+        repo = REPO_JSON_MODIFIED_SINCE.copy()
+        repo["modified_since"] = str(datetime.datetime.now().replace(tzinfo=pytz.UTC))
+        response = self.repo.process_list(api_version=1, data=repo)
+        assert tools.match(EMPTY_RESPONSE, response) is True
+
+        # without modified date
+        repo["repository_list"] = ["REPO-W/O-MODIFIED"]
+        response = self.repo.process_list(api_version=1, data=repo)
+        assert tools.match(EMPTY_RESPONSE, response) is True
```

Comment:
Both cases test only negative (empty) response. But I think it's more important to test positive response when you get some data from server. You fixed some paging feature, so I would expect adding tests where this feature is used.